### PR TITLE
SDIT-1218 Correct OverrideAccess method for /api/movements/{agencyId}/in

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java [hunk 1]
```diff
@@ -418,7 +418,7 @@ private static OffenderMovement transformToOffenderMovement(final ExternalMoveme
             .build();
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
+    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java:421]: there is no bookingId passed in - so should us VerifyAgencyAccess (for the agencyId)

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt [hunk 2]
```diff
@@ -21,89 +21,106 @@ import java.time.temporal.ChronoUnit
 import java.util.Map
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt:21]: Tidied up tests - ready for additional tests to be added (and added additional tests for agencyId access check)

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt [hunk 3]
```diff
@@ -170,8 +188,8 @@ class MovementResourceTest : ResourceTest() {
     }
 
     @Test
-    fun `should return success when does not have agency in caseload`() {
-      webTestClient.get().uri("/api/movements/SFI/in/2019-01-10")
+    fun `should return 404 when does not have agency in caseload`() {
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt:191]: corrected typo

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**